alerts/src/androidLibMain/kotlin/AlertPresenter.kt

```diff
@@ -123,7 +134,10 @@ actual class AlertPresenter(
             .applyIf(alert.style == Alert.Style.ACTION_LIST) {
                 val titles = alert.actions.map { it.title }.toTypedArray()
                 setItems(titles) { _, which ->
-                    val action = alert.actions[which].apply { handler() }
+                    val action = alert.actions[which].apply {
+                        logger.info(TAG, "Action ${this.title} was called on dialog with title: ${alert.title}")
```

Comment:
Why cant these just be done in `presentation.afterHandler()`

Reply:
Indeed, changed to that.